lib/axios.ts

```diff
@@ -4,7 +4,7 @@ import {createObservable} from './create-observable';
 
 class Axios {
 
-  static defaults: AxiosRequestConfig = axios.defaults;
+  static defaults: Omit<AxiosRequestConfig<any>, 'headers'> = axios.defaults;
```

Comment:
@Whazor Thanks for the PR. Sorry for my late reply (Somehow I didn't get any email notification about it).
Would you please explain why 'headers' should be removed?

Reply:
That is currently the type used for defaults. But maybe it is better to use [AxiosDefaults](https://github.com/axios/axios/blob/8e67551177990ed067384e1641d6964dcab773f7/index.d.ts#L126) as it would be more future proof.